sdk/core/azure-core/azure/core/pipeline/_tools_async.py

```diff
@@ -31,3 +32,36 @@ async def await_result(func, *args, **kwargs):
         # type ignore on await: https://github.com/python/mypy/issues/7587
         return await result  # type: ignore
     return result
+
+def to_rest_response(pipeline_transport_response):
+    response_type = None
+    try:
+        from .transport import AioHttpTransportResponse
+        from ..rest._aiohttp import RestAioHttpTransportResponse
+        if isinstance(pipeline_transport_response, AioHttpTransportResponse):
+            response_type = RestAioHttpTransportResponse
+    except ImportError:
+        pass
+    try:
+        from .transport import AsyncioRequestsTransportResponse
```

Comment:
Do we need to import all of them? 

Not only needed in except ImportError?

Reply:
i can add a check for each of these to only try to import if response_type is still `None`. But basically I put them each into their own try except import error block bc, for example, if you have a trio transport and no aiohttp isntalled, importing aiohttp will cause an error for you, so I wanted to accomodate all of the different combos

Reply:
ok, I now stop importing once I get a reseponse type. Thanks for the suggestion @xiangyan99 !